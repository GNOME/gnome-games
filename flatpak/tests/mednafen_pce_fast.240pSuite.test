[Retro Reftest]
Path=/mednafen_pce_fast/240pSuite
Core=/app/lib/libretro/mednafen_pce_fast_libretro.so
Medias=240pSuite.pce;

[Frame 120]
Video=mednafen_pce_fast.240pSuite.120.png
