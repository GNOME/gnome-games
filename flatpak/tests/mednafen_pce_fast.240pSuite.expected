/mednafen_pce_fast/240pSuite/Boot: OK
/mednafen_pce_fast/240pSuite/120/FastForward: OK
/mednafen_pce_fast/240pSuite/120/Run: OK
/mednafen_pce_fast/240pSuite/120/Video: OK
